### PR TITLE
vscode-extensions.streetsidesoftware.code-spell-ch: 2.3.1 -> 2.10.1

### PR DESCRIPTION
**Pull request by x10an14**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by datafoo — COMMENTED** [review]
Looks good to me.

However, why not updating to the latest version 2.10.0 (https://marketplace.visualstudio.com/items/streetsidesoftware.code-spell-checker/changelog)?

**Comment by x10an14** [comment]
> However, why not updating to the latest version 2.10.0 (https://marketplace.visualstudio.com/items/streetsidesoftware.code-spell-checker/changelog)?

I didn't check marketplace manually, I just moused over the prompt(s) I got in VS Codium.

Can fix in ~1h or so.

**Comment by datafoo** [comment]
> I didn't check marketplace manually, I just moused over the prompt(s) I got in VS Codium.
> 
> Can fix in ~1h or so.

Thank you! Can you update the  commit message and the PR message accordingly?

**Comment by x10an14** [comment]
> Thank you! Can you update the commit message and the PR message accordingly?

/derp... Ofc

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 1]
```diff
@@ -2374,8 +2374,8 @@ let
         mktplcRef = {
           name = "code-spell-checker";
           publisher = "streetsidesoftware";
-          version = "2.3.1";
-          sha256 = "0pm9i3zw4aa4qrcqnzb9bz166rl7p6nwb81m9rqzisdc85mx4s3x";
+          version = "2.10.1";
+          sha256 = "FeYkSML6QYtuIHIbAovOqlPwkKfNkHr7IdMCWwkynQ0=";
```
Comment (datafoo) [inline pkgs/applications/editors/vscode/extensions/default.nix:2378]: https://nixos.wiki/wiki/Nix_Hash

```suggestion
          sha256 = "sha256-FeYkSML6QYtuIHIbAovOqlPwkKfNkHr7IdMCWwkynQ0=";
```

**Review by datafoo — CHANGES_REQUESTED** [review]

**Comment by datafoo** [comment]
Thank you. It looks good to me :+1:

**Review by datafoo — APPROVED** [review]

**Comment by datafoo** [comment]
I did what the bot was asking and created PR https://github.com/NixOS/nixpkgs/pull/194723

**Pull request merged**